galaxy_ng/app/settings.py

```diff
@@ -402,6 +402,25 @@
 # NOTE: For the Resource Sync Feature the following are required:
 # RESOURCE_SERVER = {"URL": str, "SECRET_KEY": str, "VALIDATE_HTTPS": bool}
 
+# ANSIBLE BASE RBAC
+ANSIBLE_BASE_ROLE_CREATOR_NAME = "{obj._meta.model_name} Creator Role"
+ANSIBLE_BASE_DELETE_REQUIRE_CHANGE = False
+ANSIBLE_BASE_ALLOW_TEAM_PARENTS = False
+ANSIBLE_BASE_ALLOW_TEAM_ORG_ADMIN = False
+ANSIBLE_BASE_MANAGED_ROLE_REGISTRY = {}
+ANSIBLE_BASE_ALLOW_CUSTOM_ROLES = True
```

Comment:
https://github.com/ansible/django-ansible-base/blob/1ed57b02d5c29122c548742561d52d19e69c93c5/ansible_base/lib/dynamic_config/settings_logic.py#L180

For settings which are already the DAB default, I would prefer to not set them here. That should make no difference for the present moment, and will help make things easier to change later.

Reply:
Ok, I brought only the different settings and also the same comments from DAB